tests/specs/utils/http.spec.js

```diff
@@ -114,18 +114,22 @@ describe('HTTP', () => {
             });
         });
 
+        function randomIntBetween(min, max) {
+            return Math.floor(Math.random() * (max - min)) + min;
+        }
+
         describe('#handleStatus', () => {
-            for (let status = 200; status < 300; status++) {
+            [200, randomIntBetween(201, 299), 299].forEach((status) => {
```

Comment:
To the devil I would say: as it stands if the code were to be implemented as `if(statusCode === 200 || statusCode === 299)` unit tests would break :)

</🚲 🏠 >